theia-rust-docker/next.package.json

```diff
@@ -0,0 +1,51 @@
+{
+    "private": true,
+    "dependencies": {
+        "@theia/callhierarchy": "next",
+        "@theia/core": "next",
+        "@theia/cpp-debug": "0.3.19",
+        "@theia/editor": "next",
+        "@theia/editorconfig": "next",
+        "@theia/editor-preview": "next",
+        "@theia/file-search": "next",
+        "@theia/filesystem": "next",
+        "@theia/git": "next",        
+        "@theia/getting-started": "next",
+        "@theia/json": "next",
+        "@theia/keymaps": "next",
+        "@theia/languages": "next",
+        "@theia/markers": "next",
+        "@theia/merge-conflicts": "next",
+        "@theia/messages": "next",
+        "@theia/metrics": "next",
+        "@theia/mini-browser": "next",
+        "@theia/monaco": "next",
+        "@theia/navigator": "next",
+        "@theia/outline-view": "next",
+        "@theia/output": "next",
+        "@theia/plantuml": "next",
+        "@theia/plugin": "next",
```

Comment:
If you do not require compatibility with VS Code extensions, you could save space and build time by removing `@theia/plugin*`

Reply:
Is required. The cpp-debugging plugin depends on some vs code.

Reply:
I don't think so: @theia/cpp-debug is purely a Theia extension. Maybe the presence of a `.vscode` folder in the repo made it seem that way, but it just contains some repo-specific IDE configs that Theia can also consume, at least in good part. (in the early days of Theia, most of us were using VS Code to develop Theia)

In any case it's ok to keep the plugin-related extensions, specially if we'd like to try using the VS Code rust extension instead of the corresponding Theia extension.